rotkehlchen/globaldb/updates.py

```diff
@@ -196,33 +196,34 @@ def _parse_asset_data(self, insert_text: str) -> ParsedAssetData:
             raise DeserializationError(
                 f'At asset DB update could not parse asset data out of {insert_text}',
             )
-        if len(match.groups()) != 4:
+        if len(match.groups()) != 5:
             raise DeserializationError(
                 f'At asset DB update could not parse asset data out of {insert_text}',
             )
-        raw_type = self._parse_str(match.group(2), 'asset type', insert_text)
-        raw_started = self._parse_optional_int(match.group(3), 'started', insert_text)
+        name = self._parse_str(match.group(2), 'name', insert_text)
+        raw_type = self._parse_str(match.group(3), 'asset type', insert_text)
+        raw_started = self._parse_optional_int(match.group(4), 'started', insert_text)
         started = Timestamp(raw_started) if raw_started else None
-        swapped_for = self._parse_optional_str(match.group(4), 'swapped_for', insert_text)
+        swapped_for = self._parse_optional_str(match.group(5), 'swapped_for', insert_text)
 
         match = self.common_asset_details_re.match(insert_text)
         if match is None:
             raise DeserializationError(
                 f'At asset DB update could not parse common asset '
                 f'details data out of {insert_text}',
             )
-        forked = self._parse_optional_str(match.group(6), 'forked', insert_text)
+        forked = self._parse_optional_str(match.group(5), 'forked', insert_text)
         asset_type = AssetType.deserialize_from_db(raw_type)
 
         return ParsedAssetData(
             identifier=self._parse_str(match.group(1), 'identifier', insert_text),
             asset_type=asset_type,
-            name=self._parse_str(match.group(2), 'name', insert_text),
-            symbol=self._parse_str(match.group(3), 'symbol', insert_text),
+            name=name,
```

Comment:
I don't see `name` being used above. Why not just do...
`name=self._parse_str(match.group(2), 'name', insert_text)`